utils/common/src/test/java/brooklyn/util/stream/StreamGobblerTest.java

```diff
@@ -41,9 +43,9 @@ public void testGobbleStream() throws Exception {
         StreamGobbler gobbler = new StreamGobbler(stream, out, null);
         gobbler.start();
         try {
-            gobbler.join(10*1000);
+            gobbler.join(1000*1000);
```

Comment:
That's a very long wait? Can we make it more like 30 seconds, if that is what windows takes?  Should we make this in the `Integration` group if it's going to take many seconds?